dmapiclient/data.py

```diff
@@ -560,6 +560,18 @@ def find_draft_services(self, supplier_id, service_id=None, framework=None):
     find_draft_services_iter = make_iter_method('find_draft_services', 'services')
     find_draft_services_iter.__name__ = str("find_draft_services_iter")
 
+    def find_draft_services_by_framework(self, framework_slug, page=None, status=None, supplier_id=None):
+        params = {
+            'page': page,
+            'status': status,
+            'supplier_id': supplier_id
+        }
```

Comment:
In other endpoints we only add stuff to the dictionary if it is not None, does it matter either way?

Reply:
I did think about that at first, but for this new method I followed the style for paginated endpoints - see https://github.com/alphagov/digitalmarketplace-apiclient/blob/master/dmapiclient/data.py#L815. I don't think it makes a huge difference.